Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM python:3.9.1
+FROM python:3.10.9-bullseye
 
 EXPOSE 9808
 
-RUN apt-get update && apt install -y locales libcurl4-openssl-dev libssl-dev \
+RUN apt-get update && \
+    apt-get -y dist-upgrade && \
```

Comment:
"dist-upgrade" here to apply latest debian security updates to close critical CVEs.